shared/storage/minio.py

```diff
@@ -270,6 +291,11 @@ def read_file(
                 )
             raise e
 
+        if metadata_container is not None:
+            for key, value in response.headers.items():
+                if key.startswith("x-amz-meta-"):
+                    metadata_container[key[11:]] = value
```

Comment:
is there a `strip_prefix` kind of function? that would be a better idea rather than just slicing an number of chars off the front.